src/Api/V1/Tests/Action/CoreTest.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace Api\V1\Tests\Action;
+
+use Common\ApiTestCase;
+
+class CoreTest extends ApiTestCase
+{
+    public function testErrorOutput()
```

Comment:
I would give this a more vebose name. Something like "testApiGivesErrorWithoutParameters"